ansible/scripts/deploy-local-cluster.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Unless https://github.com/kubernetes/contrib/issues/1707 gets resolved,  "All rights reserved." has to be there.